lldb/test/API/commands/expression/import-std-module/deque-dbg-info-content/Makefile

```diff
@@ -1,5 +1,8 @@
-# FIXME: once the expression evaluator can handle std libraries with debug
-# info, change this to USE_LIBCPP=1
-USE_SYSTEM_STDLIB := 1
```

Comment:
Is this related to the linker flag changes in any way? We rely on this on the macOS bots, so it doesn't sound like the underlying issue the FIXME referred to was resolved. If it's an unrelated change, I'd prefer opening a separate PR for these